packages/api/lib/writeRecords/write-granules.js

```diff
@@ -192,10 +192,16 @@ const _writePostgresGranuleViaTransaction = async ({
     granuleRecord,
   });
 
-  log.info(`
+  if (!upsertQueryResult[0]) {
+    log.info(`
+    Did not update ${granuleRecord.granule_id}, collection_cumulus_id ${granuleRecord.collection_cumulus_id}
+    due to granule overwrite constraints, retraining original granule for cumulus_id ${pgGranule.cumulus_id}`);
```

Comment:
```suggestion
    due to granule overwrite constraints, restraining original granule for cumulus_id ${pgGranule.cumulus_id}`);
```
Should this say `restraining`?

Reply:
Otherwise, this addition is great! Thanks so much for adding it 🙇

Reply:
*retaining* actually :D

Reply:
27e92f9bc7

Reply:
Ohh `retaining` makes **a lot** more sense